roles/ceph-osd/templates/ceph-osd-run.sh.j2

```diff
@@ -70,6 +70,10 @@ fi
 # MAIN #
 ########
 
+{% if ceph_osd_numactl_opts != "" %}
+numactl \
```

Comment:
@guits Not sure if numactl is available by default. Maybe it's worth to install the associated package in the ceph-osd role.

Reply:
@dsavineau I agree and I think that's a good idea for a follow up patch. It's not included in Red Hat based distros by default. FWIW in the OpenStack case we're triggering ceph-ansible with a tag to not install any packages as we prefer to let TripleO's image have the packages pre-installed. Here's the patch to have TripleO do this: 

https://review.openstack.org/#/c/642466

Reply:
See https://github.com/ceph/ceph-ansible/pull/3705 for the follow up